src/app/views/comp/artist.html

```diff
@@ -1,10 +1,13 @@
-<div class="table" ng-repeat="artist in list">
-	<div class="artist" ng-controller="viewCtrl" ng-click="viewCtrl.changeView('/artist/' + artist.id)">
-		<div class="artist-img">
-			<img class="img" ng-src="{{artist.images[0].url}}" alt="{{artist.name}}">
+<div class="table">
+	<div class="item" ng-repeat="item in list" ng-controller="viewCtrl" ng-click="viewCtrl.changeView(type + item.id)">
```

Comment:
be careful with this, you are instantiating the controller n times, this is not performant, instead you can add the controller in the top div, that one with the `table` class